docs/dev/integrate_operator_how_to.md

```diff
@@ -0,0 +1,62 @@
+## How to add a new operator to the installer?
+
+This document describes how to provide an operator's configuration files and manifests to installer-launched clusters.
+
+One can classify all the manifest/config files of an operator in two categories:
+ - Static
+ - Templatized
+ 
+The static ones are clearly just byte blobs and the templates are ones that have some variable that needs to be filled up. Typically the variables that need filling up will come from user preferences as specified in the install config (e.g. cluster name, cluster domain, service cidr etc). Other dependencies could be TLS cert keys for example.
+
+### The recommended way
+The static and template files of your operator need to be dealt with separately.
+
+*Static files*
+
+For static files use the Cluster Version Operator (CVO) payload mechanism. There is a particular way to keep the manifest files so that the CVO update payload can pick it up.
+See this document:
+
+https://docs.google.com/document/d/1CGZVEyuloZ9oD4NUArW6dnEpi0WFc6BP2tqPSwFZTCY/edit#heading=h.1zgrwxmpgxbr
+
+Also remember that the order of creation of the files is alphabetical, so the files should be numbered like below to effectively create the service account before the deployment.
+```
+00-namaspace.yaml
+...
+03-roles.yaml
+04-serviceaccount.yaml
+05-deployment.yaml
+```
+where, 04/05 is the internal ordering of the resource manifests.
+
+*What to do for the dynamic template files?*
+
+An operator should auto-discover the install config rather than expand the templates through the installer integration (see alternative). Simply use the apiserver access to get the install-config as a config map. The config map is stored by the name ‘cluster-config-v1’ in the ‘kube-system’ namespaces.
+Psuedo code:
+```
+kube-client(apiserver-url).Get(Resource: “config-map”, Namespace: “kube-system”, Name: “cluster-config-v1”)
```

Comment:
nit: This code snippet should use straight quotes `"` ;).

Reply:
That. Because I copy pasted from a google doc :). Fixed.